### PR TITLE
Exit signup refactor

### PR DESCRIPTION
**Pull request by kingoflolz**
lets see if travis gets it right this time

### CODE REVIEW

**Comment by jkilpatr** [comment]
what's up with the tests here?

File: rita/src/rita_exit/db_client/mod.rs [hunk 1]
```diff
@@ -110,121 +108,243 @@ fn verify_identity(details: &ExitRegistrationDetails, request_ip: &IpAddr) -> Re
...
+        zip: client
+            .reg_details
+            .zip_code
+            .clone()
+            .unwrap_or("".to_string()),
+        country,
+    }
+}
+
+fn email_ver_done(_mesh_ip: &IpAddr, _conn: &SqliteConnection) -> Result<bool, Error> {
```
Comment (jkilpatr) [inline rita/src/rita_exit/db_client/mod.rs]: does this need to be finished, also where is sending the email itself?

Comment (jkilpatr) [reply rita/src/rita_exit/db_client/mod.rs]: ok so we still need integration with fastmail / some mail service

Comment (kingoflolz) [reply rita/src/rita_exit/db_client/mod.rs]: yeah, planning to do that in a new PR as it is pretty self contained (and this one is big enough as is)

**Pull request merged**